spec/abstractops.html

```diff
@@ -1909,7 +1932,7 @@ <h1>
           </tr>
           <tr>
             <td>*"offset"*</td>
-            <td>~ToString~</td>
+            <td>~ToStringIfStringOrObject~</td>
             <td>*undefined*</td>
```

Comment:
Isn't it also strange to be strict on `offset` but flexible on the other fields?